integration-libs/digital-payments/src/checkout/facade/dp-checkout-payment.service.ts

```
@@ -0,0 +1,67 @@
+import { map, catchError } from 'rxjs/operators';
+import { DpPaymentRequest } from '../models/dp-checkout.model';
+import { Observable, of } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { PaymentDetails } from '@spartacus/core';
+import { Command, CommandService, CommandStrategy } from '@spartacus/core';
+import { Query, QueryService } from '@spartacus/core';
+import { DigitalPaymentsAdapter } from '../adapters/digital-payments.adapter';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DpCheckoutPaymentService {
+  constructor(
+    protected dpAdapter: DigitalPaymentsAdapter,
+    protected command: CommandService,
+    protected query: QueryService
+  ) {}
+
+  protected RequestUrlQuery: Query<DpPaymentRequest> = this.query.create(() =>
+    this.dpAdapter.createPaymentRequest().pipe(
+      map((payload: any) => {
+        var payment_req = {
+          url: payload.postUrl,
+          sessionId: payload.parameters.entry.find(
+            (it: any) => it.key === 'session_id'
+          ).value,
+          signature: payload.parameters.entry.find(
+            (it: any) => it.key === 'signature'
+          ).value,
+        };
+        return payment_req;
+      }),
+      catchError(() => of({}))
```

What's the reason for swallowing errors here?

---

have removed this catch statement